ansible/configs/ocs4-external-implementation/pre_software.yml

```diff
@@ -32,6 +32,13 @@
 - name: Step 003.1 - Configuring Bastion Hosts
   hosts: bastions
   become: true
+  pre_tasks:
+    - name: set ocp network subnetid
+      set_fact:
+        ocp_network_subnet_id: "{{ hostvars['localhost']['ocp_network_subnet_id'] }}" 
+
+    - debug: var=ocp_network_subnet_id
+
```

Comment:
Please use consistent YAML indentation. No other YAML list has extra indent in this file.